### DISCUSSION TITLE
How to handle possible need to restore host OS from zfs_autobackup replication pull of that host

### DISCUSSION THREAD

**Discussion by hput** [psy0rz/zfs_autobackup#136] [Q&A] [answered]
I'm about to embark on a repair mission on HP-Z800 that has a sata '15/22' socket rendered unusable in one of the HDD drawers (1 of 4).  
Looks like a mechanical repair that will require some disassembly of the Tower chassis.  Followed by some kind of cobbled homeboy attempt at restoring the socked to its proper position.

I've taken the precaution of pulling a replication backup (with zfs_autobackup) of that host, onto a second HP host (xw8600).  So in the event I do something that renders the Z800 non-bootable.  I'll have the replication backup to restore it. 

But, WHOOPS, I suddenly realized I have no idea how that might be done.  Never have attempted anything like it. 
Since it is not now an immediate problem, I wanted a brief summary idea of how such a restore is done.

**Comment by psy0rz** [comment, marked as answer]
https://github.com/psy0rz/zfs_autobackup/wiki#restore-example

**Reply by hput** [reply]
Sorry for the negligence about reading that
